app/controllers/v0/institutions_controller.rb

```diff
@@ -88,5 +88,14 @@ def facets
       }
     end
     # rubocop:enable AbcSize
+
+    # Embed search result counts as a list of hashes with "name"/"count"
+    # keys so that open-ended strings such as country names do not
+    # get interpreted/mutated as JSON keys.
+    def embed(group_counts)
+      group_counts.each_with_object([]) do |(k, v), array|
+        array << { name: k, count: v }
+      end
```

Comment:
I think we can do this in the database with group style query and it will be faster, but we can revisit that later.